peripherals/pssi/pssi.yaml

```diff
@@ -0,0 +1,206 @@
+_copy:
+  PSSI:
+    from: DCMI
+
+_modify:
```

Comment:
This `_modify` block should be split into a new file, and put in `devices/common_patches/pssi/pssi.yaml` for example.